elements/reigns/src/Meter.tsx

```diff
@@ -38,10 +50,14 @@ export const Meter = ({
 
     previousPercent.current = percent;
     return () => clearTimeout(timeout);
-  }, [percent]);
+  }, [percent, phase]);
 
   return (
-    <div className={"meter"}>
+    <div className={clsx("meter",
+        phase === GamePhase.NOT_STARTED && 'meter--game-not-started',
+        phase === GamePhase.ENDED && 'meter--game-ended',
+        phase === GamePhase.STARTED && 'meter--game-started',
+      )}>
```

Comment:
`clsx` also supports:

```suggestion
    <div className={clsx("meter", {
      "meter--game-not-started": phase === GamePhase.NOT_STARTED,
      "meter--game-ended": phase === GamePhase.ENDED,
      "meter--game-started": phase === GamePhase.STARTED,
    })}>
```